site/public/js/ta-grading.js

```diff
@@ -891,3 +891,277 @@ function adjustSize(name) {
     textarea.style.height = "";
     textarea.style.height = Math.min(textarea.scrollHeight, 300) + "px";
 }
+
+function renderPDFToolbar(){
```

Comment:
Don't combine these functions into this file. Keep them in separate files. We will probably also want to display some elements of the toolbar to students (zoom, rotate) to students in the future at some point anyway.

Reply:
I've moved the renderPDFToolbar into the remade PDFInitToolbar.js file. I moved initializePDFToolbar (now loadPDFToolbar) into PDFAnnotateEmbedded.js (got an odd error when trying to place it in PDFAnnotate.js)